kernel/classes/ezcontentobjecttreenodeoperations.php

```diff
@@ -113,6 +113,7 @@ static function move( $nodeID, $newParentNodeID )
                 $nodeIDList = array( $nodeID );
                 eZSearch::removeNodeAssignment( $node->attribute( 'main_node_id' ), $newNode->attribute( 'main_node_id' ), $object->attribute( 'id' ), $nodeIDList );
                 eZSearch::addNodeAssignment( $newNode->attribute( 'main_node_id' ), $object->attribute( 'id' ), $nodeIDList, true );
+                eZSearch::refreshChildren( $node );
```

Comment:
> yes, this seems legit. @andrerom what do you think?

If there are examples already doing this, maybe.
But I'd rather prefer we force DelayedIndexing from LegacyBridge / LegacyBundle if we know we depend on it, but then either doc will need to cover that this needs to be run, OR as hinted above  ezplatformsearch needs to take this action into account here in it's included cronjob: 

https://github.com/netgen/ezplatformsearch/blob/master/cronjobs/ezplatformindexsubtree.php

@emodric Wdyt?